coffee-module/coffee-module-etcd/src/main/java/hu/icellmobilsoft/coffee/module/etcd/producer/CachedEtcdConfigSource.java

```diff
@@ -34,9 +35,10 @@
  */
 public class CachedEtcdConfigSource extends DefaultEtcdConfigSource {
```

Comment:
`FilteredEtcdConfigSource` is szerepel a feladatleírásban. Ahhoz is hasznos lenne egy ilyen megoldás a jödőre nézve. Vagy volt valami oka, hogy az nem változott?

Reply:
szerintem erre gondolt:

https://github.com/i-Cell-Mobilsoft-Open-Source/coffee/blob/feature/717-runtime-activation-of-etcd-config-sources/docs/en/migration/migration290to2100.adoc

https://github.com/i-Cell-Mobilsoft-Open-Source/coffee/blob/feature/717-runtime-activation-of-etcd-config-sources/docs/hu/migration/migration290to2100.adoc